### PR TITLE
Attempt to include SQL hint for old optimizer

### PR DESCRIPTION
**Pull request by mhines-usgs**
@kmschoep-usgs care to check this out?

### CODE REVIEW

File: gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/jsl/data/ParameterSpec.java [hunk 1]
```diff
@@ -73,6 +73,7 @@ public ParameterSpec(String stationName, ParameterCode parameterCode, SpecOption
 	public String setupTableName() {
 		StringBuilder result = new StringBuilder();
 		
+		result.append("	 /*+ OPT_PARAM('optimizer_features_enable' '11.2.0.3') */");
 		result.append("  (SELECT MEASUREMENT_DATE AS TSM_DT,");
```
Comment (kmschoep-usgs) [inline gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/jsl/data/ParameterSpec.java]: the hint should go between 'SELECT' and 'MEASUREMENT_DATE'.

File: gcmrc-services/src/main/resources/gov/usgs/cida/gcmrcservices/mb/mappers/ReachDetailMapper.xml [hunk 2]
```diff
@@ -24,6 +24,7 @@
 	</sql>
 	
 	<select id="getReachDetails" parameterType="map" resultMap="reachDetailResult">
+		/*+ OPT_PARAM('optimizer_features_enable' '11.2.0.3') */
 		SELECT
```
Comment (kmschoep-usgs) [inline gcmrc-services/src/main/resources/gov/usgs/cida/gcmrcservices/mb/mappers/ReachDetailMapper.xml]: The hint should go after this 'SELECT' and before the column list.

**Comment by mhines-usgs** [comment]
@kmschoep-usgs thanks, fixed the placement. safe to merge?

**Comment by kmschoep-usgs** [comment]
Looks good!

**Comment by mhines-usgs** [comment]
:+1:

**Pull request merged**